src/login/use-action-text.js

```diff
@@ -0,0 +1 @@
+export const useTextAction = user => user ? 'Logout' : 'Login'
```

Comment:
Would the name `useButtonValue` be more relevant here?

Reply:
Should we also rename the file into `use-button-value.js` in this case?

Reply:
Done